.travis.yml

```diff
@@ -45,6 +46,7 @@ install:
   - gem install bundler
   - bundle install
   - git clone https://github.com/DataDog/dd-agent.git ~/dd-agent
+  - cp ~/dd-agent/datadog.conf.example ~/dd-agent/datadog.conf
```

Comment:
It will still fail on `initialize_logging()` on some other checks without `datadog.conf` so, that would have to be somehow mocked too.... It's too much of a hassle for our contributors. I'm going to ensure we copy on the gem side though, to make sure we have the same `datadog.conf` used for local and travis test runs.